cms/templates/course_outline.html

```diff
@@ -31,6 +31,20 @@
     <%static:include path="js/${template_name}.underscore" />
 </script>
 % endfor
+<script>
+        function load() {
```

Comment:
A small comment or a more explicit function name may be helpful here as well. It's fairly generic and I had to reread through the logic to remember why this would have been added to the PR. If I didn't have prior context I may not be sure as to why this is needed on the page.